schema.sql

```diff
@@ -0,0 +1,7 @@
+create database if not exists pollervertx;
+
+create table if not exists endpoints (id int primary key auto_increment, name varchar(50), endpoint text, frequency int, active boolean);
+
```

Comment:
You should tell mysql to use pollervertx before creating the tables

Suggest: 

use pollervertx;

Reply:
ahh good catch! will add and push